### ISSUE TITLE
Inference code and config using base model

### ISSUE THREAD

**Issue by LeeKeyu** [SkyworkAI/Matrix-Game#46]
Dear authors, thanks for your inspiring work. Could you please add the implementation of inference using base model instead of the distilled ones? I think the provided code lacked specification of how to use the pretrained base model weight provided on huggingface.
Thanks a lot in advance!

**Comment by LeeKeyu** [comment]
您好，我尝试复现base model的效果，使用：
`python inference_streaming.py     --config_path configs/inference_yaml/inference_universal_base.yaml     --checkpoint_path Matrix-Game-2.0/base_model/diffusion_pytorch_model.safetensors     --img_path demo_images/universal/0000.png     --output_folder outputs     --seed 42     --pretrained_model_path Matrix-Game-2.0`
并把scheduler设为默认的100步，输出的图像为纯噪声。请问有哪些需要修改的地方吗？感谢！

**Comment by nik-55** [comment]
I faced similar issue in inference. However, was able to figure out. This notebook might be helpful: https://gist.github.com/nik-55/8d389f8ad1cb797a9054b6e6b689cbe3

**Comment by liushuchun** [comment]
> Thanks for your interest in Matrix-Game v2! Maybe the pre-trained VAE checkpoint was not loaded. It is supposed to check if you've downloaded the checkpoint from HuggingFace and try again.

你好，问一下base_model有启动脚本吗？这块跑出来都是噪声。
